site/src/components/WorkspaceActions/WorkspaceActions.tsx

```diff
@@ -140,6 +147,27 @@ export const WorkspaceActions: FC<WorkspaceActionsProps> = ({
             <DeleteOutlined />
             Delete
           </MenuItem>
+          <Divider sx={{ borderColor: (theme) => theme.palette.divider }} />
+
+          {isBuildLogsVisible ? (
+            <MenuItem
```

Comment:
One slightly weird thing about this menu item is that if you have an error then toggling this does nothing since we hardcode it to show in that case, which I think could feel unexpected.  Should we make it toggleable even if there is an error?  

Or remove this option when there is an error?  

Or maybe we can reword the menu option to something like "Toggle default build log visibility".

Reply:
Ahhh good catch!